game/map.moon

```diff
@@ -51,7 +51,7 @@ Map = Class "Map",
 
 		perlinMap: (arg) ->
 			self = Map arg
-			@noises or= 1 -- FIXME
+			@noises or= 1
```

Comment:
Variable name suggests several values are stored. Why is this only an integer? Does this need renaming?